remote-api/src/main/java/org/alfresco/rest/api/impl/NodesImpl.java

```diff
@@ -2971,6 +2985,14 @@ public Node upload(String parentFolderNodeId, FormData formData, Parameters para
                     renditionNames = getStringOrNull(field.getValue());
                     break;
 
+            case "versioningenabled":
+                    String versioningEnabledStringValue = getStringOrNull(field.getValue());
+                    if(null != versioningEnabledStringValue)
```

Comment:
We normally have a space after the "if"

Reply:
Thank you for spotting this, fixed it in the last commit.